riotx-android/pipeline.yml

```diff
@@ -19,9 +19,10 @@ steps:
 
   - label: "Compile Android tests"
     agents:
-      # We use a medium sized instance instead of the normal small ones because
+      # We use a xlarge sized instance instead of the normal small ones because
       # gradle build can be memory hungry
-      queue: "medium"
+      # build on the burn-after-use ephemeral queue
```

Comment:
Why? Unless we execute any of the built artifacts, we should use the regular queue,to make more efficient use of the VM